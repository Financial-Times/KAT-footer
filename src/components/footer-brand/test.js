import React from 'react';
import { expect } from 'chai';
import { mount } from 'enzyme';
import FooterBrand from './index.js';

describe('FooterBrand', () => {

  describe('component', () => {
    const wrapper = mount(<FooterBrand/>);

    const content = <div className="o-footer__brand">
      <div className="o-footer__container">
        <div className="o-footer__brand-logo"></div>
      </div>
    </div>;

    test('has the right content', () => {
      expect(wrapper.contains(content)).to.equal(true);
    });
  });
});
