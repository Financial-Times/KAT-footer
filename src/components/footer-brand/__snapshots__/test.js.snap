// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterBrand component matches snapshot 1`] = `
"<div className=\\"o-footer__brand\\">
<div className=\\"o-footer__container\\">
<div className=\\"o-footer__brand-logo\\" />
</div>
</div>"
`;
